e2e_tests/tests/cluster/test_job_queue.py

```diff
@@ -0,0 +1,84 @@
+import subprocess
+from time import sleep
+from typing import Dict, List, Tuple
+
+import pytest
+
+# from determined.experimental import Determined, ModelSortBy
+from tests import config as conf
+from tests import experiment as exp
+
+
+@pytest.mark.e2e_cpu
+def test_job_queue_adjust_weight() -> None:
+    config = conf.tutorials_path("mnist_pytorch/const.yaml")
+    model = conf.tutorials_path("mnist_pytorch")
+    for _ in range(2):
+        exp.create_experiment(config, model)
+
+    jobs = JobInfo()
+    ok = jobs.refresh_until_populated()
+    assert ok
+
+    ordered_ids = jobs.get_ids()
+    subprocess.run(["det", "job", "update", ordered_ids[0], "--weight", "10"])
+
+    sleep(2)
+    jobs.refresh()
+    new_weight = jobs.get_job_weight(ordered_ids[0])
+    assert new_weight == "10"
+
+    subprocess.run(["det", "job", "update-batch", f"{ordered_ids[1]}.weight=10"])
+
+    sleep(2)
+    jobs.refresh()
+    new_weight = jobs.get_job_weight(ordered_ids[1])
+    assert new_weight == "10"
+
+    return
+
+
+def get_raw_data() -> Tuple[List[Dict[str, str]], List[str]]:
+    data = []
+    ordered_ids = []
+    output = subprocess.check_output(["det", "job", "list"]).decode("utf-8")
```

Comment:
nit, would say just to use python bindings for the api instead, but it doesn't need to block this.